examples/src/main/python/hbase_inputformat.py

```diff
@@ -72,6 +75,9 @@
         keyConverter=keyConv,
         valueConverter=valueConv,
         conf=conf)
+    # hbase_rdd is a RDD[dict]
```

Comment:
In fact, every record in dict is a record of HBase. 
Therefore by using RDD[dict], the element of RDD corresponds a record in HBase. If we use RDD of dict, it is impossible to tell a certain value belongs to which row, which column, etc.